template/[% if repo_host_type == 'gitlab.com' or repo_host_type == 'gitlab-self-managed' %].gitlab[% endif %]/workflows/ci.yml.jinja

```diff
@@ -33,6 +33,7 @@ ci:
     - make dev
     - make lint test doc build
   stage: ci
+[% if project_name == 'Serious Scaffold Python' -%]
```

Comment:
There are many patterns to handle the empty line, but let us keep consistency here:

```
[%- if ... %]
...
[%- endif %]
```